content/docs/setup/kubernetes/platform-setup/docker-for-desktop/index.md

```diff
@@ -6,8 +6,8 @@ skip_seealso: true
 keywords: [platform-setup,kubernetes,docker-for-desktop]
 ---
 
-If you want to run istio under docker for desktop's built-in kubernetes, you may need to increase docker's memory limit
-under the *Advanced* pane of docker's preferences.  Pilot by default requests 2048Mi of memory, which is docker's
+If you want to run istio under docker for desktop's built-in Kubernetes, you may need to increase docker's memory limit
+under the *Advanced* pane of docker's preferences.  Pilot by default requests `2048Mi` of memory, which is docker's
 default limit.  Alternatively, you may reduce pilot's memory reservation by passing the helm argument
 `--set pilot.resources.requests.memory="512Mi"`.  Otherwise pilot may refuse to start due to insufficient resources.
```

Comment:
Pilot should be capitalized.

Reply:
A screenshot for the Advanced pane of Docker's preferences might help in a followup PR.